daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -49,12 +51,15 @@ spec:
     value: "true"
 ```
 
+> For details on using `secretKeyRef`, see the guide on [how to reference secrets in components]({{< ref component-secrets.md >}}).
+
 ## Spec metadata fields
 
 | Field              | Required | Details | Example |
 |--------------------|:--------:|---------|---------|
 | brokers             | Y | A comma-separated list of Kafka brokers. | `"localhost:9092,dapr-kafka.myapp.svc.cluster.local:9093"`
 | consumerGroup       | N | A kafka consumer group to listen on. Each record published to a topic is delivered to one consumer within each consumer group subscribed to the topic. | `"group1"`
+| consumerID       | N | Consumer ID (a.k.a consumer tag) organizes one or more consumers into a group. Consumers with the same consumer ID work as one virtual consumer, i.e. a message is processed only once by one of the consumers in the group. If the consumer ID is not set, the dapr runtime will set it to the dapr application ID. | `"topic1"`
```

Comment:
I would replace the "AKA" and "ie" as best to use english work. For example "or consumer tag" and "meaning that a message is processed only..."
Also dapr->Dapr and reword this sentence
"If the `consumerID` is not provided, the Dapr runtime set it to the Dapr application ID (`appID`) value."